kubejs/server_scripts/1000mek_wind_gene.js

```diff
@@ -1,100 +1,37 @@
-        'SWS',
-        'WDW',
-        'SWS'
-    ],{
-        S: 'mekanism:seismic_vibrator',
-        W: 'compactmekanismmachines:compressed_wind_generator_x512',
-        D: 'mekanism:dynamic_valve'
-    })
-
-    event.shaped('compactmekanismmachines:compressed_wind_generator_x8192', [
-        'MWM',
-        'WGW',
-        'MWM'
-    ],{
-        M: 'mekanism:thermal_evaporation_valve',
-        W: 'compactmekanismmachines:compressed_wind_generator_x2048',
-        G: `megacells:cell_component_1m`
-    })
-
-    event.shaped('compactmekanismmachines:compressed_wind_generator_x32768', [
-        'HWH',
-        'WGW',
-        'HWH'
-    ],{
-        H: 'mekanism:sps_casing',
-        W: 'compactmekanismmachines:compressed_wind_generator_x8192',
-        G: `megacells:cell_component_4m`
-    })
-
-    event.shaped('compactmekanismmachines:compressed_wind_generator_x131072', [
-        ' W ',
-        'WSW',
-        ' W '
-    ],{
-        W: 'compactmekanismmachines:compressed_wind_generator_x32768',
-        S: 'mekaevolution:absolute_energy_cube'
-    })
+    event.recipes.mekanismMetallurgicInfusing(`compactmekanismmachines:compressed_wind_generator_x2`,'2x mekanismgenerators:wind_generator', '100x mekanism:redstone')
 
-    event.shaped('compactmekanismmachines:compressed_wind_generator_x532480', [
-        ' W ',
-        'WSW',
-        ' W '
-    ],{
-        W: 'compactmekanismmachines:compressed_wind_generator_x131072',
-        S: 'compactmekanismmachines:compressed_wind_generator_x8192'
-    })
+    function addWindGeneratorRecipe(result_multipliers,item1,item2){
+        event.shaped(`compactmekanismmachines:compressed_wind_generator_x${result_multipliers}`, item2==undefined?[
+            ' W ',
+            'WHW',
+            ' W '
+        ]:[
+            'SWS',
+            'WHW',
+            'SWS'
+        ],(item2==undefined)?{
+            H: item1,
+            W: `compactmekanismmachines:compressed_wind_generator_x${result_multipliers!=532480?result_multipliers/4:131072}`,
+        }:{
+            H: item1,
+            W: `compactmekanismmachines:compressed_wind_generator_x${result_multipliers!=532480?result_multipliers/4:131072}`,
+            S: item2
+        })
+    }
+    addWindGeneratorRecipe(8,'mekanism:alloy_infused')
+    addWindGeneratorRecipe(32,'mekanism:steel_casing')
+    addWindGeneratorRecipe(128,'mekanism:basic_control_circuit','mekanism:fuelwood_heater')
+    addWindGeneratorRecipe(512,'mekanism:module_base','mekanism:module_base')
+    addWindGeneratorRecipe(2048,'mekanism:dynamic_valve','mekanism:seismic_vibrator')
+    addWindGeneratorRecipe(8192,'megacells:cell_component_1m','mekanism:thermal_evaporation_valve')
+    addWindGeneratorRecipe(32768,'megacells:cell_component_4m','mekanism:sps_casing')
+    addWindGeneratorRecipe(131072,'mekaevolution:absolute_energy_cube')
+    addWindGeneratorRecipe(532480,'compactmekanismmachines:compressed_wind_generator_x8192')
```

Comment:
値のハードコーディングを避け、代わりにmultipliersの値を使用するよう変更します。
```suggestion
    addWindGeneratorRecipe(multipliers[1], 'mekanism:alloy_infused')
    addWindGeneratorRecipe(multipliers[2], 'mekanism:steel_casing')
    addWindGeneratorRecipe(multipliers[3], 'mekanism:basic_control_circuit', 'mekanism:fuelwood_heater')
    addWindGeneratorRecipe(multipliers[4], 'mekanism:module_base', 'mekanism:module_base')
    addWindGeneratorRecipe(multipliers[5], 'mekanism:dynamic_valve', 'mekanism:seismic_vibrator')
    addWindGeneratorRecipe(multipliers[6], 'megacells:cell_component_1m', 'mekanism:thermal_evaporation_valve')
    addWindGeneratorRecipe(multipliers[7], 'megacells:cell_component_4m', 'mekanism:sps_casing')
    addWindGeneratorRecipe(multipliers[8], 'mekaevolution:absolute_energy_cube')
    addWindGeneratorRecipe(multipliers[9], 'compactmekanismmachines:compressed_wind_generator_x8192')
```